tools/gc_decode.cpp

```diff
@@ -79,8 +94,10 @@ int main(int argc, char *argv[]) {
 
   if (prontohex) {
     irsend.sendPronto(gc_test, index);
-  } else {
+  } else if (dumpgc) {
     irsend.sendGC(gc_test, index);
+  } else if (dumpraw) {
```

Comment:
`dumpraw` is used for dumping the message out in "raw" format.
e.g. for converting a GlobalCache/Pronto message into "raw" format.